functions-and-operators/Cast 函数和操作符.md

```diff
@@ -0,0 +1,13 @@
+# Cast 函数和操作符
+
+**Cast 函数和操作符**
```

Comment:
use TOC

Reply:
use upper case

Reply:
This `Cast` is not a function name,
but represents a kind of functions.